package.json

```diff
@@ -52,9 +51,11 @@
     "install:backend": "poetry install --no-root",
     "install:frontend": "cd frontend && yarn",
     "lint:frontend": "cd frontend && yarn lint",
-    "start": "electron .",
+    "start": "yarn electron .",
+    "dev": "dotenv -e .env -- yarn start",
     "start:frontend": "cd frontend && yarn start",
-    "test:frontend": "cd frontend && yarn test"
+    "test:frontend": "cd frontend && yarn test",
+    "download-binaries": "sh download_binaries.sh x64"
```

Comment:
apologies @SarthakDev12, could you please make this download both binaries?
x64 and the arm64 variant too?
fine to run curl twice here, and remove the x64 from the script

Reply:
sure thing